src/Components/ColaboraConNos/ColaboraConNos.tsx

```diff
@@ -0,0 +1,29 @@
+import { css } from "aphrodite";
+import styles from "./styles";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCamera, faLocationDot } from "@fortawesome/free-solid-svg-icons";
+
+const ColaboraConNos = () => {
```

Comment:
This component and it name's file should be something like "Colaborate" or "ColaborateComponent"